package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Backend for the famous TischSpiel",
   "main": "dist/app.js",
   "scripts": {
-    "setup": "cd scripts && setup.sh && cd ..",
+    "setup": "cd scripts && setup.sh",
```

Comment:
why wouldnt you want to go back up after executing the shell script ?

Reply:
You don't have to go back because after the command you will return to the path from where you started anyway because of the behaviour npm scripts are handled.

Nevertheless i have made a change that the npm install is now not a part from the bash script rather of the npm script. This is more secure to the circumstances that you can run the bash script without npm is installed.